Library/Formula/fish.rb

```diff
@@ -15,26 +15,19 @@ class Fish < Formula
     url "https://github.com/fish-shell/fish-shell.git", :shallow => false
 
     depends_on "autoconf" => :build
-    # Indeed, the head build always builds documentation
     depends_on "doxygen" => :build
   end
 
-  skip_clean "share/doc"
-
   def install
     system "autoconf" if build.head?
     # In Homebrew's 'superenv' sed's path will be incompatible, so
     # the correct path is passed into configure here.
     system "./configure", "--prefix=#{prefix}", "SED=/usr/bin/sed"
     system "make", "install"
-  end
 
-  def post_install
-    system "pkill fishd || true"
```

Comment:
I would like to leave this in for this release. If fishd is running and hung, then users will get the "Old instances of fish are running" message until they manually kill fishd.